src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -119,22 +120,22 @@ sub-<label>/[ses-<label>/]
-| Proton density     | PD               |                                                                                                                                                                                                                                                      |
-| Proton density map | PDmap            |                                                                                                                                                                                                                                                      |
-| Combined PD/T2     | PDT2             |                                                                                                                                                                                                                                                      |
-| Inplane T1         | inplaneT1        | T1-weighted anatomical image matched to functional acquisition                                                                                                                                                                                       |
-| Inplane T2         | inplaneT2        | T2-weighted anatomical image matched to functional acquisition                                                                                                                                                                                       |
-| Angiography        | angio            |                                                                                                                                                                                                                                                      |
+| Name               | `modality_label` | Description                                                                                                                                       |
+| ------------------------- | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| T1 weighted        | T1w              |                                                                                                                                                   |
+| T2 weighted        | T2w              |                                                                                                                                                   |
+| T1 Rho map         | T1rho            | Quantitative T1rho brain imaging <br> <https://www.ncbi.nlm.nih.gov/pubmed/24474423> <br> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346383/> |
```

Comment:
Why did you change the rendering of these links? If the links break the pandoc process, then I would find it acceptable to put the links as simple text in there 

... what I don't get are your added `<` and `>` characters

Reply:
So, I hadn't made any changes here. But in the commit before this, `src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md` was failing `TravisCI` tests. However, the fix mentioned here: https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md worked. In that, `TravisCI` tests weren't failing anymore but I didn't really go into `src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md` to see what had done the trick. 

But just figured that this was caused by running `remark` on this file.

Reply:
Although, having angle brackets doesn't affect how its rendered on RTD or pdf. So for now, until further feedback, the most recent commit has the angle brackets.